frontend/build.sh

```diff
@@ -7,13 +7,13 @@ esbuild                                          \
     --minify                                     \
     "--define:process.env.NODE_ENV='production'" \
     --pure:console.log
-cd dist
+cd dist || exit 1
 js_hash="main.$(md5sum main.js | head -c 10).js"
 mv main.js "$js_hash"
 css_hash="main.$(md5sum main.css | head -c 10).css"
 mv main.css "$css_hash"
-cat ../public/index.html |
-    sed 's,.*main.js.*,<script src="'"/$js_hash"'"></script>,' |
-    sed 's,<head>,&<link href="'"/$css_hash"'" rel="stylesheet">,' |
-    cat > index.html
+
+sed -e 's,.*main.js.*,<script src="'"/$js_hash"'"></script>,' \
+    -e 's,<head>,&<link href="'"/$css_hash"'" rel="stylesheet">,' \
```

Comment:
It's a _tiy_ bit odd that the `/` in front of the shell variable expansions aren't part of the single-quoted `sed -e` string, but instead in the double quoted string, i.e. that it is `'..."'"/$var"'"...'` instead of `'..."/'"$var"'"...'`.